### PR TITLE
added synonym to MONDO:0013886

### LINKED ISSUE #7358: CAMTA1-related disorder
### Mondo term

cerebellar dysfunction with variable cognitive and behavioral abnormalities

### Synonyms

CAMTA1-related disorder

### Synonym type

exact

### Additional comments

Hi!  We had a request from a patient who was diagnosed with this condition.  Thank you! Gioconda
https://pubmed.ncbi.nlm.nih.gov/38044714/

### PR DESCRIPTION
**Pull request by sabrinatoro**
Closes #7358

### CODE REVIEW

File: src/ontology/mondo-edit.obo [hunk 1]
```diff
@@ -303765,14 +303765,14 @@ subset: nord_rare {source="MONDO:NORD"}
...
-synonym: "cerebellar ataxia, nonprogressive, with mental retardation" RELATED DEPRECATED [MONDO:Lexical, OMIM:614756]
+synonym: "CAMTA1-related disorder" EXACT [https://orcid.org/0000-0001-9310-0163, PMID:38044714]
+synonym: "CANPMR" EXACT DEPRECATED [MONDO:Lexical, OMIM:614756]
+synonym: "cerebellar ataxia, nonprogressive, with intellectual disability" EXACT [MONDO:Lexical, OMIM:614756]
+synonym: "cerebellar ataxia, nonprogressive, with mental retardation" EXACT DEPRECATED [MONDO:Lexical, OMIM:614756]
 synonym: "cerebellar dysfunction with variable cognitive and behavioral abnormalities" EXACT [OMIM:614756, OMIM:genemap2]
-synonym: "non-progressive cerebellar ataxia with intellectual disability" RELATED [Orphanet:314647]
-synonym: "nonprogressive cerebellar ataxia with intellectual disability" RELATED [DOID:0050998]
-synonym: "nonprogressive cerebellar ataxia with mental retardation" RELATED DEPRECATED [DOID:0050998]
-synonym: "nonprogressive cerebellar atxia with intellectual disability" EXACT CLINGEN_LABEL []
```
Comment (twhetzel) [inline src/ontology/mondo-edit.obo:303775]: Nice catch on the typo, e.g. `atxia`!

File: src/ontology/mondo-edit.obo [hunk 2]
```diff
@@ -303765,14 +303765,14 @@ subset: nord_rare {source="MONDO:NORD"}
...
+synonym: "non-progressive cerebellar ataxia with intellectual disability" EXACT [Orphanet:314647]
+synonym: "nonprogressive cerebellar ataxia with intellectual disability" EXACT CLINGEN_LABEL [DOID:0050998, Orphanet:314647]
```
Comment (twhetzel) [inline src/ontology/mondo-edit.obo:303774]: I don't see that Orphanet:314647 lists "nonprogressive cerebellar ataxia with intellectual disability" only the version with the hyphen.

**Review by twhetzel — COMMENTED** [review]
question inline on the synonym source

File: src/ontology/mondo-edit.obo (hunk 2 above)
Comment (sabrinatoro) [reply src/ontology/mondo-edit.obo:303774]: done

**Review by twhetzel — APPROVED** [review]
lgtm

**Pull request merged**